### PR TITLE
Released Stateful Pipeline to Production

### PR DESCRIPTION
**Pull request by victorskl**

### CODE REVIEW

**Comment by victorskl** [comment]
Note: I have deploy Shared Stack at this point as per instruction.

File: lib/pipeline/statefulPipelineStack.ts [hunk 1]
```diff
@@ -122,24 +122,24 @@ export class StatefulPipelineStack extends cdk.Stack {
 
     // Some stack have dependencies to the 'shared stack' so we need to deploy it first beforehand
```
Comment (williamputraintan) [inline lib/pipeline/statefulPipelineStack.ts:123]: I guess we could remove this comment now

**Review by williamputraintan — APPROVED** [review]
LGTM!

File: lib/pipeline/statefulPipelineStack.ts (hunk 1 above)
Comment (victorskl) [reply lib/pipeline/statefulPipelineStack.ts:123]: Similar thought. 
Let keep the comment there for future init another environment/account.
This comment is good.

**Pull request merged**